lib/elixir/test/elixir/task_test.exs

```diff
@@ -272,6 +272,139 @@ defmodule TaskTest do
     end
   end
 
+  @tag timeout: 10
```

Comment:
This will set the timeout to 10 milliseconds, which is most likely not what you want. The default is already 30 seconds, you can safely remove it.